x-pack/plugins/remote_clusters/public/sections/remote_cluster_list/remote_cluster_table/remote_cluster_table.test.js

```diff
@@ -53,4 +60,45 @@ describe('RemoteClusterTable', () => {
 
     expect(component.find('tbody > tr').first()).toMatchSnapshot();
   });
+
+  describe('row actions', () => {
+    const name = 'test-cluster';
+
+    const clusters = [{
+      name,
+      seeds: ['seed'],
+    }];
+
+    test('name link opens detail panel when clicked', () => {
+      const component = mountWithIntl(
```

Comment:
I would have probably put the component instantiation inside a `beforeEach` to avoid duplication in both tests.

Reply:
I think it's incidental that they happen to instantiate with identical state, so I'd prefer to avoid the coupling introduced by a `beforeEach` here.

Reply:
On second thought, because we're testing row actions it makes sense to couple the tests with a `beforeEach` and in fact, they're already coupled by the `clusters` variable.